OrangeHRMTests/PageObjects/Pages/AdminPage.cs

```diff
@@ -0,0 +1,66 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.Extensions.ExtensionMethods;
+using OrangeHRMTests.Common.WebElements;
+using OrangeHRMTests.PageObjects.Modules;
+
+namespace OrangeHRMTests.PageObjects.Pages
+{
+    public class AdminPage : BasePage
+    {
+        public MyWebElement SystemUserTableHeaderElement = new MyWebElement(By.XPath("//*[contains(@class, 'table-filter')]//*[contains(@class, 'filter-title')]"));
+        private MyWebElement DropdownElement = new MyWebElement(By.XPath("//ul[@class='oxd-dropdown-menu']//*[contains(@role, 'menuitem')]"));
```

Comment:
you create MyWebElement althouth you have DropDownExtension.. hmm

Reply:
I've created it because it is quite unique for Admin page and is used only here